### DISCUSSION TITLE
per-ingester local limit: 150000

### DISCUSSION THREAD

**Discussion by FengZh61** [grafana/mimir#2443] [Help and support]
level=warn ts=2022-07-18T07:09:24.348726256Z caller=grpc_logging.go:38 method=/cortex.Ingester/Push duration=1.313085ms err="rpc error: code = Code(400) desc = user=anonymous: per-user series limit of 150000 exceeded, please contact administrator to raise it (per-ingester local limit: 150000)" msg=gRPC
level=error ts=2022-07-18T07:09:24.349252342Z caller=push.go:89 user=anonymous msg="push error" err="rpc error: code = Code(400) desc = user=anonymous: per-user series limit of 150000 exceeded, please contact administrator to raise it (per-ingester local limit: 150000)"

Logs frequently refresh such logs

This is my yaml configuration file

multitenancy_enabled: false

blocks_storage:
  backend: filesystem
  bucket_store:
    sync_dir: /tmp/mimir/tsdb-sync
  filesystem:
    dir: /tmp/mimir/data/tsdb
  tsdb:
    dir: /tmp/mimir/tsdb

compactor:
  data_dir: /tmp/mimir/compactor
  sharding_ring:
    kvstore:
      store: memberlist

distributor:
  ring:
    instance_addr: 127.0.0.1
    kvstore:
      store: memberlist

ingester:
  ring:
    instance_addr: 127.0.0.1
    kvstore:
      store: memberlist
    replication_factor: 1

ruler_storage:
  backend: local
  local:
    directory: /tmp/mimir/rules

server:
  http_listen_port: 9009
  log_level: info

store_gateway:
  sharding_ring:
    replication_factor: 1

**Comment by pracucci** [comment]
Hi! Please look at this runbook, which addresses the error you're getting:
https://grafana.com/docs/mimir/next/operators-guide/mimir-runbooks/#err-mimir-max-series-per-user

_Note: in Mimir 2.2.0 (will be released soon) we've improved error messages and they will include the unique error ID you can lookup in the [errors catalog](https://grafana.com/docs/mimir/next/operators-guide/mimir-runbooks/#errors-catalog)._
